content/docs/user-guide/project-structure/dvcignore-files.md

```diff
@@ -91,9 +91,12 @@ b026324c6904b2a9cb4b88d6d61c81d1  data/data1
 26ab0db90d72e28ad0ba1e22ee510510  data/data2
 ```
 
-Only the cache entries of the `data/` directory itself and one file have been
-stored. Checking the hash value of the data files manually, we can see that
-`data2` was cached. This means that `dvc add` did ignore `data1`.
+There are only 2 cache entries, and one of them (the one starting with `ad` and
+ending in `.dir`) is for the `data/` directory itself. Checking the hash value
+of the data files manually, we can see that the other cache entry (the one
+starting with `26`) is for `data2`. There is no cache entry for the `data1` file
+(whose hash value starts with `ab`). This means that `dvc add` did ignore
+`data1`.
```

Comment:
TMI, I think. This doc is not about cache structure. But we could link to https://dvc.org/doc/user-guide/project-structure/internal-files#structure-of-the-cache-directory if needed.

Was there a specific reason for this update?